.rubocop.yml

```diff
@@ -1,3 +1,5 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
Thanks for doing this. I may actually pull this onto our [custom rubocop](https://github.com/angellist/rubocop-angellist) ruleset we use here Angellist eventually, but this is a net improvement and I really appreciate the effort!